### PR TITLE
IA-732 font awesome

### PR DESCRIPTION
**Pull request by beygorghor**
- remove css, scss, from react app
- use scss only for django templates
- replace map css removed from blsq component (clusters and custom zomm control)
- replace custom zoom controls icons (box and fit to bound) with svg icons
-  ignore svg in tests

### CODE REVIEW

**Comment by beygorghor** [comment]
⚠️ this PR is using https://github.com/BLSQ/bluesquare-components/pull/7

File: hat/templates/iaso/login.html [hunk 1]
```diff
@@ -13,7 +13,8 @@ <h1 class="auth__heading"> <img alt="iaso logo" src="{% static 'images/logo.png'
         {% if next %}
         {% if user.is_authenticated %}
         <p class="auth__text--warning">
-          {% blocktrans %}Your account doesn't have access to this page. To proceed, please login with an account that has access.
+          {% blocktrans %}Your account doesn't have access to this page. To proceed, please login with an account that
```
Comment (olethanh) [inline hat/templates/iaso/login.html:16]: please check that it doesn't break translation

File: hat/templates/iaso/login.html [hunk 2]
```diff
@@ -32,7 +33,8 @@ <h1 class="auth__heading"> <img alt="iaso logo" src="{% static 'images/logo.png'
         <input type="hidden" name="next" value="{{ next }}" />
         <button type="submit" class="button">{% trans 'Login' %}</button>
         <button id="display-password">
-          <i class="fa fa-eye" id="eye-icon" aria-hidden="true"></i>
+          {% include "./icons/eye.html" %}
+          {% include "./icons/eye-slash.html" %}
```
Comment (olethanh) [inline hat/templates/iaso/login.html:37]: not sure this is the cleanest way compared to including the icon as static? but I guess this is probably less prone to failure

Comment (beygorghor) [reply hat/templates/iaso/login.html:37]: yes initially i put svg directly in the template, if you have a better suggestion

File: hat/templates/iaso/login.html (hunk 1 above)
Comment (beygorghor) [reply hat/templates/iaso/login.html:16]: this is a mistake

Comment (beygorghor) [reply hat/templates/iaso/login.html:16]: mistake, i'll put back old code there

File: hat/assets/js/apps/Iaso/index.js [hunk 3]
```diff
@@ -11,6 +11,7 @@ import ProtectedRoute from './domains/users/components/ProtectedRoute';
 import { store, history } from './redux/store';
 import { addRoutes } from './routing/redirections';
 import { getPlugins, PluginsContext } from './utils';
+import { getOverridedTheme } from './styles';
```
Comment (quang-le) [inline hat/assets/js/apps/Iaso/index.js:14]: should be `getOverriddenTheme`

**Review by quang-le — APPROVED** [review]

**Review by olethanh — APPROVED** [review]

**Pull request merged**